onto-viewer-core/src/test/java/org/edmcouncil/spec/ontoviewer/core/ontology/data/handler/ModuleHandlerTest.java

```diff
@@ -62,18 +63,17 @@ void shouldReturnListOfModulesDefinedInOntology() {
 
   @Test
   void shouldReturnListOfModulesWhenNotDefinedInOntology() {
-    var moduleHandler = prepareModuleHandler("/ontology/MortgageLoansWithoutImports.rdf");
-
+    if (automaticCreationOfModules) {
```

Comment:
I don't think that you need to add this check here.

Reply:
I added because unit tests do not passed.

Reply:
Yeah, but this isn't a good way to solve the problem with this unit test. You simply switched it off here. You should modify this in the configuration that is passed to `ModuleHandler`. I see that you don't use the application-wide `ApplicationConfigurationService` in `ModuleHandler` - you should also do it, because for now you use hardcoded value.

Reply:
Okey, I'll change it.